### PR TITLE
Pre 27 sns s3 utils

### PR DESCRIPTION
**Pull request by adberglund**
This util work can facilitate the story requirements for #27

### CODE REVIEW

File: cloudigrade/util/aws.py [hunk 1]
```diff
@@ -220,3 +222,88 @@ def _verify_policy_action(session, action):
...
+
+    object_bytes = s3_object.get()['Body'].read()
+
+    if compression == 'gzip':
+        content = gzip.decompress(object_bytes).decode('utf-8')
+    elif compression is None:
+        try:
+            content = object_bytes.decode('utf-8')
+        except UnicodeDecodeError as e:
+            logger.error(_(e))
```
Comment (ilyavw) [inline cloudigrade/util/aws.py]: Not sure if we can honestly expect gettext to know what to do with an exception message. ¯\_(ツ)_/¯

Comment (infinitewarp) [reply cloudigrade/util/aws.py]: @ilyavw I think your shrug guy dropped this: \\

Comment (adberglund) [reply cloudigrade/util/aws.py]: I'm not even being consistent, either as right above this I am decoding without try/except. In keeping with our "very minimal to no error handling" I could just expect it to be UTF-8 at this point? Alternatively I can replace with `logger.error(_('Content is not UTF-8 encoded'))` and duplicate the try/except above this.

Comment (ilyavw) [reply cloudigrade/util/aws.py]: Quite a bit of work goes into fixing that and the lean `¯\\\_(ツ)\_/¯`

Comment (ilyavw) [reply cloudigrade/util/aws.py]: Maybe throw in a small test for that branch while you're at it? ;)

**Pull request merged**